### PR TITLE
Improve deployment script

### PR DESCRIPTION
**Pull request by fregante**
While using the workflow, I found some improvements to be made. I'll keep this PR open to keep track of them and bring them back here.

❗️ ❗️ ❗️  Depends on these 2 packages being released as v2:
- https://github.com/actions/upload-artifact/releases
- https://github.com/actions/download-artifact/releases

### CODE REVIEW

File: .github/workflows/deploy-tag.yml [hunk 1]
```diff
@@ -33,9 +33,8 @@ jobs:
     steps:
     - uses: actions/download-artifact@v1
       with:
-        name: built-extension
-        path: distribution
-    - run: cd distribution && npx chrome-webstore-upload-cli upload --auto-publish
+        name: distribution
```
Comment (notlmn) [inline .github/workflows/deploy-tag.yml]: This should be `path`

```suggestion
        path: distribution
```

Comment (fregante) [reply .github/workflows/deploy-tag.yml]: No, path is automatically the same name as the artifact

https://github.com/actions/download-artifact#usage

Edit: but indeed the other step was wrong

File: .github/workflows/deploy-tag.yml [hunk 2]
```diff
@@ -48,9 +47,8 @@ jobs:
     steps:
     - uses: actions/download-artifact@v1
       with:
-        name: built-extension
         path: distribution
```
Comment (fregante) [inline .github/workflows/deploy-tag.yml]:
```suggestion
        name: distribution
```

File: .github/workflows/deploy-tag.yml (hunk 1 above)
Comment (fregante) [reply .github/workflows/deploy-tag.yml]: When `download-artifact@v2` is available we can drop the name too:

https://github.com/actions/download-artifact/blob/3bdf740f024d67bba81315347e81ea6450d53b5b/src/download-artifact.ts#L11-L13

And also in `upload-artifact@v2`

https://github.com/actions/upload-artifact/blob/40fe78158e409ef7de32764afadffce845677f8c/src/constants.ts#L6-L8

**Comment by fregante** [comment]
https://github.blog/changelog/2020-04-28-github-actions-v2-artifact-actions/ 🎉 

Will be tested with #28

**Pull request merged**